Arriven/db1000n
# Update Windows EC2 instance

# Description

- Replaced Windows AMI with OpenSSH server installed 
- Copy public key to the Windows server
- Reset password upon new Windows EC2 instance spin up

## Type of change


## How Has This Been Tested?

On existing AWS account:
1. Ensure Windows instance doesn't exist
2. Run ansible-playbook aws-provisioning.yaml --tags windows
3. Wait for about 15-20 minutes
4. Ensure password is retrievable
5. Ensure you can log on to the server using SSH and private key

## Test Configuration

- Release version:
- Platform: Ubuntu 20.04


## Screenshots

- [x] No screenshot
